k8s/operators/src/pool/diskpool/crd/v1beta3.rs

```diff
@@ -28,7 +28,9 @@ use super::quantity::Quantity;
     printcolumn = r#"{ "name":"encrypted", "type":"boolean", "description":"encryption enabled", "jsonPath":".status.encrypted"}"#,
     printcolumn = r#"{ "name":"capacity", "type":"string", "nullable": "true", "description":"total bytes", "jsonPath":".status.capacity_q"}"#,
     printcolumn = r#"{ "name":"used", "type":"string", "nullable": "true", "description":"used bytes", "jsonPath":".status.used_q"}"#,
-    printcolumn = r#"{ "name":"available", "type":"string", "nullable": "true", "description":"available bytes", "jsonPath":".status.available_q"}"#
+    printcolumn = r#"{ "name":"available", "type":"string", "nullable": "true", "description":"available bytes", "jsonPath":".status.available_q"}"#,
+    printcolumn = r#"{ "name":"disk_capacity", "type":"string", "nullable": "true", "description":"underlying disk capacity", "jsonPath":".status.diskCapacity"}"#,
```

Comment:
should it be camel case here as well? Don't know what's typical for the pretty columns